doc/shell/shell_guide.md

```diff
@@ -0,0 +1,844 @@
+|dispatch_tc_ALL.txt|dispatch_tc_ALL.txt|Records all the cases that are needed to be test this time.|
+|dispatch_tc_FIN_local.txt|dispatch_tc_FIN_106.txt<br>(dispatch_tc_FIN_107.txt<br>...)|Records the cases that have been finished on this machine by now.|
+|monitor_local.log|monitor_106.log<br>(monitor_107.log<br>...)|Records the monitor logs, such as memory, disk space, processes, CUBRID logs, database, CUBRID conf files, netstat. This kind of file is only used when we set enable_status_trace_yn=true in configuration file.|
+|test_local.log|test_106.log<br>(test_107.log<br>...) |Records the screen output of CTP tool. It contains the sceen output of each test case.|
+
+
+
+## 2.3 Execute a single test case  
+Before execute a single case, we need export $init_path variable.   
+Usually, we add it in ~/.bash_profile file:    
+```
+export init_path=$HOME/CTP/shell/init_path
+```
+To execute a single test case, we just need to login a test machine, go to the case path, and then execute shell command:  
+```bash
+sh case_name.sh
+ ```
+
+
+
+# 3 Regression Test Deployment
+## 3.1 Test Machines
+|Role|User|IP|Hostname|
+|---|---|---|---|
+|controller node|shell_ctrl|192.168.1.104|func29|
+|worker node|shell|192.168.1.104|func29|
+|worker node|shell|192.168.1.105|func30|
+|worker node|shell|192.168.1.106|func31|
+|worker node|shell|192.168.1.107|func32|
+|worker node|shell|192.168.1.108|func33|
+|worker node|shell|192.168.1.109|func34|
+|worker node|shell|192.168.1.110|func35|
+|worker node|shell|192.168.1.111|func36|
+|worker node|shell|192.168.1.112|func37|
+|worker node|shell|192.168.1.113|func38|
+|worker node|shell|192.168.1.114|func39|
+
+## 3.2 create and set users  
+### controller node
+We need create a new user: shell_ctrl.
+Login root user and execute:  
+```
+sudo useradd shell_ctrl
+```
+Set password as our common password for user shell_ctrl.  
+```
+sudo passwd shell_ctrl
+```
+ Set the user's password to never expire.  
+ ```
+ sudo chage -E 2999-1-1 -m 0 -M 99999 shell_ctrl
+ ```
+ 
+### worker nodes
+We need create a new user: shell.  
+Login root user and execute:  
+```
+sudo useradd shell
+```
+Set password as our common password for user shell.  
+```
+sudo passwd shell
+```
+ Set user's password to never expire.  
+ ```
+ sudo chage -E 2999-1-1 -m 0 -M 99999 shell
+ ```
+## 3.3 install software packages
+Required software packages: jdk, lcov, bc.   
+
+|software|version|usage|  
+|---|---|---|  
+|jdk|1.8.0 (need larger than 1.6)|run CTP, run shell test case|  
+|lcov|lcov-1.11|run code coverage test|  
+|bc|latest version|run shell test case|  
+
+
+## 3.4 Deploy controller node
+### Install CTP   
+Please refer to ["CTP Installation Guide"](https://github.com/CUBRID/cubrid-testtools/blob/develop/doc/ctp_install_guide.md#2-install-ctp-as-regression-test-in-linux-platform)

```

Comment:
Change link to `https://github.com/CUBRID/cubrid-testtools/blob/develop/doc/ctp_install_guide.md#3-install-ctp-as-regression-test-platform`